episodes/infrastructure.md

```diff
@@ -196,12 +196,16 @@ You may need to wait a few minutes for the website to be generated.
 
 The lesson title can be adjusted by modifying the `config.yaml` file in the repository.
 The `config.yaml` file contains several global parameters for a lesson -
-to determine some of the page styling, contact details for the lesson, etc -
-and is written in _[YAML]_, a structured file format of key-pair values.
+to determine some of the page styling, contact details for the lesson, etc. 
+`config.yaml` is written in _[YAML]_, a structured file format of key-value pairs in the form `key: value`.
 As well as the title of the lesson,
 you can and should adjust some of the other values in `config.yaml`,
 but you should not need to add new values or learn a lot about YAML
-to be able to configure your lesson.
+to be able to configure your lesson. You should also only have to add or modify single-line 
+string values and not deal with multi-line strings or other data types (e.g. numbers, booleans, lists, dictionaries) 
+that YAML supports. YAML generally does not require quotes (both single (') or double (") are allowed) around string values unless 
+strings contain [reserved characters](https://www.w3schools.io/file/yaml-escape-characters/), though it may be good practice 
+from readability point of view to use quotes if your string contains several words.
```

Comment:
The template `config.yaml` quotes all string values, I think as a "safety first" approach to avoid unexpected issues with the reserved characters mentioned in your explanation. I think we should encourage that, rather than emphasise that quoting is optional, e.g.

```suggestion
that YAML supports. 
The template `config.yaml` aims to guide users through examples and annotations
on how to format the values they provide
e.g. if you are modifying a value wrapped in quotation marks in the template file,
it is safest to replace it with another value within quotation marks.
([Details about when values need to be quoted in YAML](https://www.w3schools.io/file/yaml-escape-characters/).)
```

Reply:
I had a look at our https://github.com/carpentries/lesson-development-training/blob/main/config.yaml when I was doing the PR and it had a mix of quoted and non-quoted strings - perhaps the most up to date one uses all quoted string @tobyhodges. I agree that we should encourage that practice through examples in the template rather than explain things.